RAGchain/benchmark/answer/metrics.py

```diff
@@ -106,3 +106,19 @@ def __init__(self):
     def retrieval_metric_function(self, knowledge: List[str], pred: str) -> float:
         score = self._token_f1_score(pred, "\n".join(knowledge))
         return score
+
+class METEOR(BasePassageAnswerMetric):
+    def __init__(self):
+        super().__init__()
+        self._metric_name = "METEOR"
+        try:
+            from evaluate import load
+        except ImportError:
+            raise ImportError("Please install huggingface dataset. pip install datasets")
```

Comment:
We have huggingface datasets library at setup.py dependency list.
You can delete this line.

Reply:
my mistake. METEOR metric exist in evaluate library. I will delete this line and add evaluate library to setup.py dependency list